tests/test_stat.py

```diff
@@ -0,0 +1,19 @@
+from queue import Queue
+from morphocut import Pipeline
+
+import pytest
+
+from morphocut import Pipeline
+from morphocut.stat import (
+    ExponentialSmoothing
+)
+
+def test_ExponentialSmoothing():
+    with Pipeline() as pipeline:
+        value = 5
+        alpha = 0.8
+        result = ExponentialSmoothing(value, alpha)
```

Comment:
This is rather uninteresting. Please provide at least two values, so you can check that the result is as expected (first value unchanged, second adjusted to lie in between).

Reply:
I didn't mean multiple alphas but multiple values that are processed. Sorry for not being more clear.